### PR TITLE
Put ccall to lrs_init within __init__

### LINKED ISSUE #28: Is ccall to `lrs_init` in the LRSLib.jl file executed every time?
Another question:

Is this command https://github.com/JuliaPolyhedra/LRSLib.jl/blob/5d819d4a52554dfd69ef26881ad59b15f191e0bd/src/LRSLib.jl#L29-L31 executed every time when `LRSLib` is imported?

Yes for sure when the package is recompiled, but it seems No upon the second time on:

```jl
julia> using LRSLib

julia> unsafe_load(cglobal((:lrs_ofp, LRSLib.liblrs), Ptr{Cvoid}))
Ptr{Nothing} @0x0000000000000000  # <- Null pointer?
```

### PR DESCRIPTION
**Pull request by oyamad**
Close #28

### CODE REVIEW

**Comment by blegat** [comment]
Doesn't it work to put the `__init__` function in `LRSLib.jl` ?

**Comment by oyamad** [comment]
I didn't try that. I guess it is logical to include it in `deps.jl`, isn't it?

**Comment by oyamad** [comment]
`fclose` added to `initmatrix(filename)` dba5608 (despite the comments in #27).

File: src/matrix.jl [hunk 1]
```diff
@@ -19,6 +19,12 @@ function initmatrix(filename::AbstractString)
     if !ok
         error("Invalid file $filename")
     end
+
+    # From lrs_close
+    ccall(:printf, Cint, (Ptr{Cchar},), "\n")
```
Comment (blegat) [inline src/matrix.jl]: Why is this needed ?

**Comment by blegat** [comment]
> I guess it is logical to include it in `deps.jl`, isn't it?

I have never seen it in `deps.jl`. As we might also transition to BinaryBuilder at some point, it is easy to keep `deps.jl` as simple as possible IMO.

File: src/matrix.jl (hunk 1 above)
Comment (oyamad) [reply src/matrix.jl]: Without it (i.e., with the current code on `master`), you have

```jl
julia> inem1 = LRSLib.HMatrix("simplex.ine");


julia> exit()
*Input taken from file simplex.ine(Terminal prompt)
```

I could have written

```jl
Libc.flush_cstdio()
println()
```

I don't know why one needs to flush `stdout` though.

**Comment by oyamad** [comment]
OK, makes sense. I will put it in `LRSLib.jl`.

**Comment by oyamad** [comment]
The `__init__` function is now put in `LRSLib.jl`.
